src/app/lib/utilities/ichefURL/index.js

```diff
@@ -33,7 +33,7 @@ const buildIChefURL = ({
     'news',
     isPng ? 'png' : null,
     resolution,
-    originCode,
+    !locator.includes(originCode) ? originCode : null,
```

Comment:
Some CPS assets coming through the CAF had the `originCode` value within the `locator` field. The existing CPS data transformation logic may have been doing stuff to mitigate this, so I just added a simple check here to fix the problem.